### DISCUSSION TITLE
Make a publicly available overview of current 'Sponsored' features/ideas

### DISCUSSION THREAD

**Discussion by colemahatmccreary** [directus/directus#15543] [Feature Requests]
Wouldn't need to share anything about who's paying, but pinning a general sponsored docket somewhere in github with links to PRs/Issues would be cool.

Might incentivize the community to help clear the backlog/contribute towards specific items where interests align.

**Comment by rijkvanzanten** [comment]
<img width="407" alt="CleanShot 2022-09-12 at 17 56 12@2x" src="https://user-images.githubusercontent.com/9141017/189765631-d85e2bff-cc6b-4a34-a0ef-c2b9f0dfa0e9.png">

Like that? 🙂

**Comment by colemahatmccreary** [comment]
I guess what I'm thinking of wouldnt be limited to sponsored stuff alone (so maybe bad title on my part), but moreso a high level overview of different priorities and ideas.

Maybe a kanban board (project?), with lists organized around feature/idea category and cards pointing to relevant discussions/issues/PRs (each of which may or may not have the sponsored badge you screenshot). Not sure how flexible those GH features are but just an idea. 

How do you guys stay organized internally? For instance when tracking similar (or redundant) issues/ideas across threads, etc? Especially with thousands of discussions/issues i'm just curious how you try to triage/keep sane?

**Reply by rijkvanzanten** [reply]
Ah gotcha. Yeah we do have an internal bug tracker project board (you can actually see it in that screenshot above called ⭐ Core), but we'll have to do quite a bit of cleanup before we can make that public 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
